src/providers/utils.ts

```diff
@@ -8,6 +7,6 @@ export function getDomainFromUrl(url: string) {
   return url.replace(/^(.*\/\/[^\/?#]*).*$/, '$1');
 }
 
-export function makeAssetUrl(serviceUrl: string = DEFAULT_SERVICE_URL, assetId: string, ks: string = '') {
+export function makeAssetUrl(serviceUrl: string = OVPConfiguration.get().serviceUrl, assetId: string, ks: string = '') {
```

Comment:
get()?

Reply:
https://github.com/kaltura/playkit-js-providers/blob/master/src/k-provider/ovp/config.js#L21

Reply:
I think we shouldn't expose the config statically like this as in the future we would probably need to refactor providers to be instances.
Can we make this config available though the provider instance the plugin gets?

Reply:
im checking

Reply:
@OrenMe the providers are already instances as far as i get it or that i dont got you right
the plugin dose not get any provider instance, the only relationship between the plugin and the provider is that it inherit from OVPProvider so we can add it as a member on the OVPProvider but not really understand the purpose since i assume the default config does not vary from insurance to insurance

Reply:
you can see you ask this from the static path `const {OVPConfiguration} = KalturaPlayer.providers;` so it is obviously not

Reply:
ok got you  - this is the providers **module**  i thought i meant to the providers themselves

Reply:
@OrenMe Anyway the  playkit-js-kaltura-cuepoints dose not use or create an instant of OVPProvider at all, so even if we would put thc configuration on the instance it could  not help

Reply:
@JonathanTGold the provider instance is available to the plugin via `this.player.provider`

Reply:
@RoyBregman do we need a default value for serviceUrl ? I see that both calls for makeAssetUrl pass a value